corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -119,6 +123,21 @@ def process_changes(self, since, forever):
                             self._record_checkpoint_in_datadog()
                         self._record_change_success_in_datadog(change)
                     self._record_change_in_datadog(change, timer)
+
+                    if timer.duration > self.slow_threshold:
+                        extra = change.metadata.to_json() if change.metadata else {}
+                        extra.update({
+                            'duration': '{} seconds'.format(timer.duration),
+                            'threshold': self.slow_threshold,
+                        })
+                        client.captureMessage(
+                            u'{} change slow to process'.format(self.pillow_id),
+                            extra=extra,
+                            time_spent=int(timer.duration * 1000),  # Convert to milliseconds
+                            fingerprint=[self.pillow_id],
```

Comment:
re billing every event that we send to sentry is billed unless if get's filtered on their side (we don't have any filters set up though) or it's over our configured rate limits.